src/vm.c

```diff
@@ -663,6 +684,23 @@ V7_PRIVATE int is_prototype_of(val_t o, val_t p) {
   return 0;
 }
 
+static val_t Std_eval(struct v7 *v7, val_t t, val_t args) {
+  val_t res = V7_UNDEFINED, arg = v7_array_at(v7, args, 0);
+  (void) t;
+  if (arg != V7_UNDEFINED) {
+    char buf[100], *p;
+    p = v7_to_json(v7, arg, buf, sizeof(buf));
```

Comment:
I'd stick with v7_to_json() cause it doesn't have a restrictio on buffer
size

On Thu, Jan 15, 2015 at 9:14 PM, Marko Mikulicic <notifications@github.com>
wrote:

> In src/vm.c
> <https://github.com/cesanta/v7/pull/156#discussion-diff-23044130>:
>
> > @@ -663,6 +684,23 @@ V7_PRIVATE int is_prototype_of(val_t o, val_t p) {
> >    return 0;
> >  }
> >
> > +static val_t Std_eval(struct v7 *v7, val_t t, val_t args) {
> > +  val_t res = V7_UNDEFINED, arg = v7_array_at(v7, args, 0);
> > +  (void) t;
> > +  if (arg != V7_UNDEFINED) {
> > +    char buf[100], *p;
> > +    p = v7_to_json(v7, arg, buf, sizeof(buf));
>
> You could use v7_stringify_value for that. (Which I guess needs some more
> love)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cesanta/v7/pull/156/files#r23044130>.
>